File: documentation/user/en/use/api/example/create-new-entity.cs
evita.UpdateCatalog(
	"testCatalog",
	session => {
		session.CreateNewEntity("Product")
			.SetAttribute(
				"name", new CultureInfo("en"),
				"ASUS Vivobook 16 X1605EA-MB044W Indie Black"
			)
			.SetAttribute(
				"name", new CultureInfo("de"),
				"ASUS Vivobook 16 X1605EA-MB044W Indie Schwarz"
			)
			.SetAttribute("catalogCode", "X1605EA-MB044W")
			.SetAttribute("stockQuantity", 1)
			.SetAssociatedData(
				"gallery",
				new string[] {
					"https://cdn.alza.cz/ImgW.ashx?fd=f4&cd=NA579p8e0&i=1.jpg",
					"https://cdn.alza.cz/ImgW.ashx?fd=FotoAddOrig&cd=NA579p8e0-04&i=1.jpg"
				}
			)
			.SetPrice(
				1, "basic", new Currency("EUR"),
				345.9m, 22m, 422m,
				true
			)
			.SetPrice(
				2, "basic", new Currency("USD"),
				365.57m, 22m, 446m,
				true
			)
			.SetReference("brand", "Brand", Cardinality.ZeroOrMore, 3)
			.SetReference("categories", "Category", Cardinality.ZeroOrMore, 3)
			.UpsertVia(session);
	}
);

File: documentation/user/en/use/api/example/create-new-entity.java
evita.updateCatalog(
	"evita",
	session -> {
		session.createNewEntity("Product")
			.setAttribute(
				"name", Locale.ENGLISH,
				"ASUS Vivobook 16 X1605EA-MB044W Indie Black"
			)
			.setAttribute(
				"name", Locale.GERMAN,
				"ASUS Vivobook 16 X1605EA-MB044W Indie Schwarz"
			)
			.setAttribute("catalogCode", "X1605EA-MB044W")
			.setAttribute("stockQuantity", 1)
			.setAssociatedData(
				"gallery",
				new String[] {
					"https://cdn.alza.cz/ImgW.ashx?fd=f4&cd=NA579p8e0&i=1.jpg",
					"https://cdn.alza.cz/ImgW.ashx?fd=FotoAddOrig&cd=NA579p8e0-04&i=1.jpg"
				}
			)
			.setPrice(
				1, "basic", Currency.getInstance("EUR"),
				new BigDecimal("345.9"), new BigDecimal("22"), new BigDecimal(422),
				true
			)
			.setPrice(
				2, "basic", Currency.getInstance("USD"),
				new BigDecimal("365.57"), new BigDecimal("22"), new BigDecimal(446),
				true
			)
			.setReference("brand", "Brand", Cardinality.ZERO_OR_ONE, 3)
			.setReference("categories", "Category", Cardinality.ZERO_OR_MORE, 3)
			.upsertVia(session);
	}
);
